internal/config/config.go

```diff
@@ -44,3 +46,15 @@ func InitConfig() {
 
 	setupDefaults()
 }
+
+// GetStruct returns the config as a plain struct. You should call this after InitConfig.
+func GetStruct() cfgmodel.FriendlyStripeSync {
+	cfg := cfgmodel.FriendlyStripeSync{}
+
+	err := viper.Unmarshal(&cfg)
+	if err != nil {
+		panic("Failed to unmarshal config: " + err.Error())
```

Comment:
NIT: return the error instead of panic?

Reply:
I figured this should never fail unless the typing of the struct is wrong 🤔 - and it saves us from handling this in a few places

If it turns out it does fail in other cases I'll revisit and make this return an err